crates/indexer/src/models/v2_objects.rs

```diff
@@ -157,4 +152,44 @@ impl Object {
             Ok(None)
         }
     }
+
+    /// This is actually not great because object owner can change. The best we can do now though
```

Comment:
What's the performance implication for doing this multiple times in the same processing logic? Is this cached somewhere? This can potentially lead to a lot more quota/capacity consumed when running processors on the client side against a grpc streaming service